tools/install-source/InstallSourcesLib/InstallSourcesLib.csproj

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="utf-8"?>

```

Comment:
any reason for a new .dll ? i.e. can't that just be compiled inside the existing tool ?